.travis.yml

```diff
@@ -39,6 +39,7 @@ jobs:
       env:
         # Add the directory where the Choco packages go
         - PATH=/c/Python35:/c/Python35/Scripts:$PATH
+        - PYTHON=python.exe
```

Comment:
I think there's a typo in the commit message "Choco's Python only provides a`python3.exe` executable". Probably not worth fixing unless you're rewriting history for another reason.